### PR TITLE
CLOUD-1112 Add configure.py script to customize deployment

### PR DESCRIPTION
**Pull request by csovant**
Jira issue? CLOUD-1112
Release 6.5.0 backport required? no
6.5.0 doc changes needed? no
7.0.0 doc changes needed? no
Required README updates made? no

Same code as PR #555 without unexpected changes on helm and samples files.

### CODE REVIEW

File: cicd/forgeops-tests/tests/postcommit/ds/test_dj_postcommit.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2019 ForgeRock AS. Use of this source code is subject to the
+# Common Development and Distribution License (CDDL) that can be found in the LICENSE file
+
+"""
+Verify the snapshot version of ForgeRock Product.
```
Comment (andrewvinall) [inline cicd/forgeops-tests/tests/postcommit/ds/test_dj_postcommit.py]: Incorrect docstring here and elsewhere.

File: cicd/forgeops-tests/config/ProductConfig.py [hunk 2]
```diff
@@ -45,7 +45,10 @@ def tests_domain():
 
 
 def base_url():
```
Comment (andrewvinall) [inline cicd/forgeops-tests/config/ProductConfig.py]: The contents here seems outdated.

File: cicd/forgeops-tests/configure.py [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+OPENIG_HELM_SUBFOLDER = "openig"
+OPENIG_IMAGE_NAME = "ig"
+OPENIG_DEPENDENCIES = ""
+
+if __name__ == "__main__":
+    # root_dir : computed using relative position from this file
+    current_dir = os.path.abspath(os.path.dirname(__file__))
+    os.chdir(os.path.join(os.path.abspath(os.path.dirname(__file__)), '..', '..'))
+    root_dir = os.getcwd()
+    os.chdir(current_dir)
```
Comment (andrewvinall) [inline cicd/forgeops-tests/configure.py]: Are you sure we want to change the current working directory after computing root__dir?

File: cicd/forgeops-tests/lib/utils/pod.py [hunk 4]
```diff
@@ -7,7 +7,6 @@
 # Lib imports
 import os
 from configparser import ConfigParser
-import multiset
```
Comment (andrewvinall) [inline cicd/forgeops-tests/lib/utils/pod.py]: Thanks for removing this.

File: cicd/forgeops-tests/tests/postcommit/ds/test_dj_postcommit.py (hunk 1 above)
Comment (csovant) [reply cicd/forgeops-tests/tests/postcommit/ds/test_dj_postcommit.py]: Fixed

File: cicd/forgeops-tests/config/ProductConfig.py (hunk 2 above)
Comment (csovant) [reply cicd/forgeops-tests/config/ProductConfig.py]: Fixed

File: cicd/forgeops-tests/configure.py (hunk 3 above)
Comment (csovant) [reply cicd/forgeops-tests/configure.py]: To have full path instead of having path with `../../`

Comment (csovant) [reply cicd/forgeops-tests/configure.py]: Guillaume helped me to find a better solution

**Pull request merged**